db/migrate/20230515210900_create_annotation_references.rb

```diff
@@ -0,0 +1,36 @@
+class CreateAnnotationReferences < ActiveRecord::Migration[5.2]
+  disable_ddl_transaction! if System::Database.postgres?
+
+  def change
+    if System::Database.mysql?
+      options = "CHARSET=utf8mb4 COLLATE=utf8mb4_bin"
+    end
+
+    create_table :annotations, options: options do |t|
+      t.string :name, limit: 42, null: false
```

Comment:
So far we only support a singe annotation `managed`. Why do we need so long annotation names? If we add more annotations, then we can always extend it. Presently it is mostly to prevent some app logic issue to leak into the database.

Why 42? It was as good as any other reasonable number. It was the first that came to my mind. Totally random, I promise. It was an answer to something. I surely knew what was the question but as soon as I've figured out the answer, I forgot the question. Weird, isn't it?

Reply:
Are you the second best computer in universe?